Installation/include/CGAL/license/MDS_3.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2016  GeometryFactory SARL (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org)
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: LGPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s) : Andreas Fabri
+//
+// Warning: this file is generated, see include/CGAL/licence/README.md
+
+#ifndef CGAL_LICENSE_MDS_3_H
+#define CGAL_LICENSE_MDS_3_H
+
+#include <CGAL/config.h>
+#include <CGAL/license.h>
+
+#ifdef CGAL_MDS_3_COMMERCIAL_LICENSE
+
+#  if CGAL_MDS_3_COMMERCIAL_LICENSE < CGAL_RELEASE_DATE
+
+#    if defined(CGAL_LICENSE_WARNING)
+
+       CGAL_pragma_warning("Your commercial license for CGAL does not cover "
+                           "this release of the 3D Mesh Data Structure package.")
+#    endif
+
+#    ifdef CGAL_LICENSE_ERROR
+#      error "Your commercial license for CGAL does not cover this release \
+              of the 3D Mesh Data Structure package. \
+              You get this error, as you defined CGAL_LICENSE_ERROR."
+#    endif // CGAL_LICENSE_ERROR
+
+#  endif // CGAL_MDS_3_COMMERCIAL_LICENSE < CGAL_RELEASE_DATE
+
+#else // no CGAL_MDS_3_COMMERCIAL_LICENSE
+
+#  if defined(CGAL_LICENSE_WARNING)
+     CGAL_pragma_warning("\nThe macro CGAL_MDS_3_COMMERCIAL_LICENSE "
+                          "is not defined."
+                          "\nYou use the CGAL 3D Mesh Generation package under "
+                          "the terms of the GPLv3+.")
```

Comment:
This file `Installation/include/CGAL/license/MDS_3.h` does not seem generated using @sloriot's script, but is probably a copy-paste of Mesh_3's one. See https://github.com/CGAL/cgal/blob/a0d93515a9c911f80e25718ac168dc3980ba57e5/Installation/include/CGAL/license/README.md.